exchange/bitswap/bitswap.go

```diff
@@ -208,6 +208,12 @@ func (bs *Bitswap) GetBlocks(ctx context.Context, keys []key.Key) (<-chan *block
 		log.Event(ctx, "Bitswap.GetBlockRequest.Start", &k)
 	}
 
+	go func() {
```

Comment:
i'm generally against timeouts in places where they should be implemented higher up by a caller.